commands/web/phpstan

```diff
@@ -18,4 +18,15 @@ test -e phpstan.neon || curl -OL https://git.drupalcode.org/project/gitlab_templ
 sed -i 's/BASELINE_PLACEHOLDER/phpstan-baseline.neon/g' phpstan.neon
 # Add an empty baseline file to ensure it exists.
 test -e phpstan-baseline.neon || touch phpstan-baseline.neon
-phpstan analyse $DDEV_DOCROOT/$DRUPAL_PROJECTS_PATH "$@"
+
+CONTRIB_MODULE_DIR=$(find "$DDEV_DOCROOT/$DRUPAL_PROJECTS_PATH" -mindepth 1 -maxdepth 1 -type d | head -n 1)
+if [ -n "$CONTRIB_MODULE_DIR" ]; then
```

Comment:
Since we now support themes, lets call this variable EXTENSION_DIRECTORY or similar

Reply:
Good idea, one my next todo item was further generalizing these command to work for Recipe contrib development with bundled code via https://www.drupal.org/project/recipe_code_installer :speak_no_evil:

Reply:
I had not seen that project before. I understand the need, but I'm still not sure its a good idea to bypass the module distribution system. Lots of think build on it (Packagist, security team, this project, etc.). This is in fact one of my hesitations about recipes in general. Lets discuss elsewhere.